salt/metalk8s/defaults.yaml

```diff
@@ -51,4 +54,38 @@ kubelet:
     authorization:
       mode: AlwaysAllow
 
+ca:
```

Comment:
We may want to put all this in a `pki` key maybe.

Reply:
The thing is that I want to have cert information in the section of the service but for this it's the `ca` stuff is for ... kubernetes so.. maybe add it to a kubernetes section, I don't know

Reply:
Leave it as it is then.